Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -135,10 +136,11 @@ public extension CBPurchase {
         self.purchaseProductHandler(product: product, completion: handler)
     }
     
-    func restorePurchases(includeInActiveProducts:Bool = false ,completion handler: @escaping ((_ result: Result<[InAppSubscription], RestoreError>) -> Void)) {
+    func restorePurchases(includeInActiveProducts:Bool = false, customer: CBCustomer, completion handler: @escaping ((_ result: Result<[InAppSubscription], RestoreError>) -> Void)) {
```

Comment:
Can we make the customer as optional?

Reply:
Please update the versions in corresponding places as well.

Reply:
> Can we make the customer as optional?

Now customerId is mandatory know then why it should be optional ?

Reply:
> Please update the versions in corresponding places as well.

ok

Reply:
@cb-gadagoju During the purchase flow, the customer object is still optional. And the same goes for validateReceipt as well. Do you think the same can be extended to the restore flow?

Reply:
@cb-haripriyan  yes we can implement optional as purchase flow but when its optional for restore merchant might consider optional and may skip it know any how he will get success after restore even if they didn't pass customer_id ? is't there any hidden problem which customer will know later again if they miss it ? i think we should highlight to customer for passing customerObject ! what you suggest ?